SU-SWS/stanford_benefits
# 7.x 1.0 alpha4 fixes

# READY FOR REVIEW

# Summary
- Fixing so ratings only show for the pre-retiree plans.

# Needed By (Date)
- 09/26/2019

# Urgency
- 7/10

# Steps to Test

1. Currently if you pick retirees and compare plans, it shows the star ratings for NCQA.
2. This fix turns that off for retirees and turns it on for pre-retirees. 
3. Go to `benefits-rewards/health/medical-plans/comparison-tool`
4. Make sure that the NCQA ratings only show for the pre-retirees when comparing plans.


# Affected Projects or Products
- `cardinalatwork` site

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/CAW-867

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)…te folders
